packages/core/package.json

```diff
@@ -38,7 +35,6 @@
     "index.js",
     "index.d.ts",
```

Comment:
Do we still need this (and `index.js`)?

Reply:
nope :).

Reply:
See https://github.com/strongloop/loopback-next/pull/945#discussion_r165288862